src/js/youtube-embed.js

```diff
@@ -20,32 +20,33 @@
 	 * and set up objects representing the videos.
 	 */
 	window.onYouTubeIframeAPIReady = function() {
-		$('.inline-youtube-video').each(function(){
-			var video_id = $(this).data('video-id'),
-				video_height = $(this).data('video-height'),
-				video_width = $(this).data('video-width');
+		$( ".inline-youtube-video" ).each( function() {
+			var video_id = $( this ).data( "video-id" ),
+				video_height = $( this ).data( "video-height" ),
+				video_width = $( this ).data( "video-width" ),
+				player;
 
-			player = new YT.Player( 'youtube-video-' + video_id, {
+			player = new window.YT.Player( "youtube-video-" + video_id, {
 				height: video_height,
 				width: video_width,
 				videoId: video_id,
 				playerVars: {
 					modestbranding: 1,
 					showinfo: 0,
-					controls: 0,
+					controls: 1,
```

Comment:
It is indeed, but I confess that it was an executive decision I made after noticing that some of the videos on the site were 10+ minutes long. Let me double check with Charles to make sure he's cool with it.